Dockerfile.arm

```diff
@@ -0,0 +1,13 @@
+FROM resin/armel-debian:jessie
```

Comment:
I'd prefer we use `armel/debian` as then seems to be the more "standard" image.  @luxas What is the primary difference with the raspbian image?  This is just for the installer image, the one that lays out the partitions and copies the kernel to install from ISO or upgrade.  Basically both of these flows don't currently work on RPi.  Currently we are working on a [Estuary D02](http://open-estuary.org/d02-2/) which is a very PC-like ARM board that support UEFI/grub so it works a lot like an intel server.  So I'm not sure what the raspbian image would provide for us, we'd prefer this to be generic ARM as possible.